docs/data-sources/node.md

```diff
@@ -50,32 +50,32 @@ data "wallarm_node" "example" {
 
 ## Argument Reference
 
-`filter` - (Required) Filters set in the `key=value` format used to look up for WAF node details. Possible keys:
+`filter` - (**required**) filters set in the `key=value` format used to look up for Wallarm node details. Possible keys:
 
-- `uuid` - (Optional) WAF node UUID.
-- `hostname` - (Optional) WAF node name.
-- `type` - (Optional) WAF node type. Can be: `cloud_node` for cloud WAF nodes, `node` for regular WAF nodes.
-- `enabled` - (Optional) Indicator of the WAF node status. Can be: `true` for enabled WAF nodes and `false` for disabled WAF nodes.
+- `uuid` - (optional) Wallarm node UUID.
+- `hostname` - (optional) Wallarm node name.
+- `type` - (optional) Wallarm node type. Can be: `cloud_node` for cloud Wallarm nodes, `node` for regular Wallarm nodes.
+- `enabled` - (optional) indicator of the Wallarm node status. Can be: `true` for enabled Wallarm nodes and `false` for disabled Wallarm nodes.
```

Comment:
some descriptions start with lower-case letter after (required/optional) and others - with upper-case, it is inconsistent

Reply:
still inconsistent, but it is up to you - to do smth with it or not